riscv/processor.cc

```diff
@@ -533,6 +533,16 @@ void state_t::reset(processor_t* const proc, reg_t max_isa)
   if (proc->extension_enabled(EXT_ZCMT))
     csrmap[CSR_JVT] = jvt = std::make_shared<jvt_csr_t>(proc, CSR_JVT, 0);
 
+  if (proc->extension_enabled(EXT_ZICFISS)) {
+    csrmap[CSR_SSP] = ssp = std::make_shared<basic_csr_t>(proc, CSR_SSP, 0);
+    csrmap[CSR_MENVCFG] = menvcfg
+      = std::make_shared<envcfg_csr_t>(proc, CSR_MENVCFG, menvcfg->read(), MENVCFG_SSE);
+    if (proc->extension_enabled('S')) {
+      csrmap[CSR_SENVCFG] = senvcfg
+        = std::make_shared<envcfg_csr_t>(proc, CSR_SENVCFG, senvcfg->read(), SENVCFG_SSE);
+    }
```

Comment:
It's not great to construct `menvcfg` and `senvcfg` again like this. It would be better to patch the original construction of these CSRs to consider this extension.